cross-chain/arbitrum/hardhat.config.ts

```diff
@@ -0,0 +1,119 @@
+import type { HardhatUserConfig } from "hardhat/config"
+
+import "@keep-network/hardhat-helpers"
+import "@nomiclabs/hardhat-waffle"
+import "@nomiclabs/hardhat-etherscan"
+import "hardhat-gas-reporter"
+import "hardhat-contract-sizer"
+import "hardhat-deploy"
+import "@typechain/hardhat"
+import "hardhat-dependency-compiler"
+
+const config: HardhatUserConfig = {
+  solidity: {
+    compilers: [
+      {
+        version: "0.8.17",
+        settings: {
+          optimizer: {
+            enabled: true,
+            runs: 1000,
+          },
+        },
+      },
+    ],
+  },
+
+  paths: {
+    artifacts: "./build",
+  },
+
+  networks: {
+    hardhat: {
+      deploy: ["deploy_l2"],
+    },
+    goerliEthereum: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 5,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    mainnetEthereum: {
+      url: process.env.CHAIN_L1_API_URL || "",
+      chainId: 1,
+      deploy: ["deploy_l1"],
+      accounts: process.env.ACCOUNTS_L1_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L1_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["etherscan"],
+    },
+    goerliArbitrum: {
+      url: process.env.CHAIN_L2_API_URL || "",
+      chainId: 421613,
+      deploy: ["deploy_l2"],
+      accounts: process.env.ACCOUNTS_L2_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L2_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["arbiscan"],
+      companionNetworks: {
+        l1: "goerliEthereum",
+      },
+    },
+    mainnetArbitrum: {
+      url: process.env.CHAIN_L2_API_URL || "",
+      chainId: 42161,
+      deploy: ["deploy_l2"],
+      accounts: process.env.ACCOUNTS_L2_PRIVATE_KEYS
+        ? process.env.ACCOUNTS_L2_PRIVATE_KEYS.split(",")
+        : undefined,
+      tags: ["arbiscan"],
+      companionNetworks: {
+        l1: "mainnetEthereum",
+      },
+    },
```

Comment:
Could we simplify the network names, like in the example below? The names for ethereum networks will still be matching the ones we're using in other ethereum-specific packages.

```ts
    goerli: {
      url: process.env.CHAIN_L1_API_URL || "",
      chainId: 5,
      deploy: ["deploy_l1"],
      ...
    },
    mainnet: {
      url: process.env.CHAIN_L1_API_URL || "",
      chainId: 1,
      deploy: ["deploy_l1"],
      ...
    },
    arbitrumGoerli: {
      url: process.env.CHAIN_L2_API_URL || "",
      deploy: ["deploy_l2"],
      companionNetworks: {
        l1: "goerli",
      },
    },
    arbitrum: {
      url: process.env.CHAIN_L2_API_URL || "",
      deploy: ["deploy_l2"],
      companionNetworks: {
        l1: "mainnet",
      },
    },
```

Reply:
Ok, here's a change [9b1b5c7](https://github.com/keep-network/tbtc-v2/pull/554/commits/9b1b5c7f5a3e297110db5310fb1aaeadbbd6750a)

The only thing that I left is `arbitrumMainnet`. I think having a "mainnet" in the name helps with identifying right off the bat what network is used.